template/meta_refresh.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+    <meta http-equiv="refresh" content="3; url=https://hub.momconnect.co.za/confirmredirect/{{pk}}?whatsappid={{whatsappid}}" />
```

Comment:
I think we can go all the way to `0` for the redirect time, to redirect instantly.

Reply:
We also don't want this URL to be hardcoded